packaging/linux/systemd/keybase.gui.service

```diff
@@ -4,8 +4,7 @@ Wants=keybase.service kbfs.service
 
 [Service]
 Type=simple
-ExecStartPre=/usr/bin/env mkdir -p %h/.cache/keybase
-ExecStart=/usr/bin/env bash -c "/opt/keybase/Keybase &>> %h/.cache/keybase/Keybase.app.log"
+ExecStart=/opt/keybase/Keybase
```

Comment:
No need for the ExecStartPre since we create the log dir before dumping

Reply:
random thought...will this be affected by the electron bug that makes us redirect output to avoid a black screen?

Reply:
That sounds like a scary bug. Stdout is affecting what gets drawn on the screen?!

Reply:
It's super weird: https://github.com/electron/electron/issues/12850

Reply:
Okay, tested that it still works.